config/ethereum.json

```diff
@@ -1,13 +1,15 @@
 {
-  "network": "ethereum",
+  "network": "mainnet",
```

Comment:
Prefer "ethereum" as found in `@airswap/constants` unless "mainnet" is required here.

Reply:
This is required